src/gradio_i18n/i18n.py

```diff
@@ -7,7 +7,8 @@
 import gradio as gr
 import langcodes
 import yaml
-from gradio.blocks import Block, BlockContext, Context, LocalContext
+from gradio.blocks import Block, BlockContext, Context
+from gradio.context import LocalContext
```

Comment:
This will make it incompatible with older versions of gradio. You could also try to import it from `context` and except an `ImportError` to import it from the old location or even clever, check for gradio's version number but this is outside the scope of this PR. I just want to translate my app 😅.